modules/caching-materials-manager-node/test/sha512.test.ts

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"). You may not use
+ * this file except in compliance with the License. A copy of the License is
+ * located at
+ *
+ *     http://aws.amazon.com/apache2.0/
+ *
+ * or in the "license" file accompanying this file. This file is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+ * implied. See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/* eslint-env mocha */
+
+import { expect } from 'chai'
+import 'mocha'
+import { sha512 } from '../src/sha512'
+
+// sha512('asdf')
```

Comment:
Should this be base64(sha512('asdf'))?

Reply:
You are not wrong.  I left the base64 off because the important part was the 'asdf'